core/app/mailers/spree/carton_mailer.rb

```diff
@@ -2,11 +2,21 @@ module Spree
   class CartonMailer < BaseMailer
     # Send an email to customers to notify that an individual carton has been
     # shipped.
-    def shipped_email(carton_id, resend: false)
-      @carton = Spree::Carton.find(carton_id)
-      subject = (resend ? "[#{Spree.t(:resend).upcase}] " : '')
-      subject += "#{Spree::Store.current.name} #{Spree.t('shipment_mailer.shipped_email.subject')} ##{@carton.order_numbers.join(', ')}"
-      mail(to: @carton.order_emails, from: from_address(Spree::Store.current), subject: subject)
+    def shipped_email(*args)
```

Comment:
or perhaps just extracting the carton & order as options instead of expecting them to be the first & second args?

this is probably just personal preference so ignore if you'd like.